admin-ui/app/components/bootstrap/bootstrap.html

```diff
@@ -7,8 +7,8 @@
         <label>
           <input type="checkbox" ng-model="bootstrap.types.android">
               <span class="ups-has-icon ups-variant-android">
-                <strong>Android</strong><br>
-                using Firebase Cloud Messaging
+                <strong translate>Android</strong><br>
```

Comment:
Does this `translate` here mean that _Android_ will be translated? In that case, is it necessary?